obspy/io/nordic/tests/data/dos-file.sfile

```diff
@@ -1,45 +1,45 @@
- 1990 1213 1109 19.8 LE 60.328   5.167  0.0F BER  6 1.3 5.9CBER                1

```

Comment:
Did you mean to change this file?  This was supposed to be used for testing reading of DOS formatted text files.

Reply:
No, this shouldn't have been changed.  If you can reject this change, please do so.  If not, please tell me how to revert to the old one.

Reply:
No, that was a mistake.  Can you reject it, or tell me how to revert to the original version?